scripts/migrate-attachments.pl

```diff
@@ -75,21 +92,38 @@
   my ($source, $dest) = @{$options{migrate}};
 
   # Do not migrate from database to net storage if data is less than minsize
-  my $where;
-  if ($source eq 'database' && $dest eq 's3' && Bugzilla->params->{attachment_s3_minsize}) {
-    $where = 'WHERE attach_size > ' . int Bugzilla->params->{attachment_s3_minsize};
+  my $where = '';
```

Comment:
Nit: since this is now just the attach size clause, we could probably rename this variable.

Reply:
Latest revision changes the script to reuse SQL in more places so this variable has been renamed as part of that.